addons/event/models/event_event.py

```diff
@@ -674,6 +681,10 @@ def copy_data(self, default=None):
         vals_list = super().copy_data(default=default)
         return [dict(vals, name=self.env._("%s (copy)", event.name)) for event, vals in zip(self, vals_list)]
 
+    def _get_event_url(self):
```

Comment:
Can't you us the field directly ? Seems there are a bit too much layers just for an overridable url ?